lib/webhook_topic_validator.rb

```diff
@@ -0,0 +1,62 @@
+require 'active_support/concern'
+module ShopifyApp
+  module WebhookTopicValidator
+    class InvalidTopic < StandardError; end
+    extend ActiveSupport::Concern
```

Comment:
nitpick but can you swap the order of the two lines and have a space in between them